templates/test/ci/cluster-template-prow-windows.yaml

```diff
@@ -161,9 +161,8 @@ metadata:
   namespace: default
 spec:
   clusterName: ${CLUSTER_NAME}
-  replicas: ${WORKER_MACHINE_COUNT}
-  selector:
-    matchLabels: null
+  replicas: "0"
```

Comment:
this fails because of https://github.com/kubernetes-sigs/kustomize/issues/3424 upgrading kustomize to 3.10 fixes it

Reply:
Upgrading to kustomize 3.10 breaks other things.  Looking into it

Reply:
fixed in #1422